source/main.lua

```diff
@@ -285,147 +394,47 @@ function playdate.update ()
         grounded = false
         jumpSound:play()
     end
 
     -- y axis kinematics
-    if gameState == playing then jumpKinematics() end
+    jumpKinematics()
 
     -- cant fire a projectile if you just did or if you are in the air
-    if gameState == playing and playdate.buttonJustPressed(playdate.kButtonA) and not isProjectileFired and grounded then
+    if playdate.buttonJustPressed(playdate.kButtonA) and not isProjectileFired and grounded then
         projectileSprite:moveTo(45, 150)
         projectileSprite:add()
         isProjectileFired = true
         shootSound:play()
     end
 
-    -- refresh screen
-    gfx.sprite.update()
-
-    -- title screen
-    if gameState == title then
-        drawTitle()
-        
-        -- switch between char positions in name
-        if playdate.buttonJustPressed(playdate.kButtonLeft) and nameIndex > 1 then
-            nameIndex -= 1
-        end
-        if playdate.buttonJustPressed(playdate.kButtonRight) and nameIndex < 3 then
-            nameIndex += 1
-        end
-
-        -- update the actual character
-        if playdate.buttonJustPressed(playdate.kButtonUp) then
-            if nameIndex == 1 then
-                l1 += 1
-                if l1 > 90 then l1 = 65 end
-            elseif nameIndex == 2 then
-                l2 += 1
-                if l2 > 90 then l2 = 65 end
-            elseif nameIndex == 3 then
-                l3 += 1
-                if l3 > 90 then l3 = 65 end
-            end
-        end
-        if playdate.buttonJustPressed(playdate.kButtonDown) then
-            if nameIndex == 1 then
-                l1 -= 1
-                if l1 < 65 then l1 = 90 end
-            elseif nameIndex == 2 then
-                l2 -= 1
-                if l2 < 65 then l2 = 90 end
-            elseif nameIndex == 3 then
-                l3 -= 1
-                if l3 < 65 then l3 = 90 end
-            end
-        end
-
-        -- form name from character
-        name = string.char(l1) .. string.char(l2) .. string.char(l3)
-
-        if playdate.buttonJustPressed(playdate.kButtonA) then
-            gameState = start
-        end
-    end
-
-    if gameState == playing then
-        gfx.drawText(score, 5, 5)
-        gfx.drawText("Like the game? Join Software Dev. Club today!", 25, 185)
-        gfx.drawText("https://discord.gg/Pvv2Eu8FrF", 80, 210)
-
-        -- obstacle movement and collision detection
-        for i, obstacleSprite in pairs(groundObstacles) do
-            -- move the obstacles towards the player
-            obstacleSprite:moveBy(baseGroundObstacleSpeed + addedGroundObstacleSpeed, 0)
-
-            -- if the obstacle is onscreen, render it
-            if obstacleSprite.x >= -7 and obstacleSprite.x <= 407 then
-                obstacleSprite:add()
-            -- otherwise, remove it
-            elseif obstacleSprite.x <= -7 then
-                obstacleSprite:remove()
-            end
+    gfx.drawText(score, 5, 5)
```

Comment:
If the player pauses or loses then this text won't be displayed. Since these 3 lines should always be displayed unless the player is at the title screen then you can just move then up higher in the function (before where line 381 currently is should work).

Reply:
should be fixed now